### PR TITLE
Docs for Windows Home support and QSG

### PR DESCRIPTION
**Pull request by usha-mandya**
- Added a new topic to describe the pre-reqs and  installation instructions for Windows Home.
- Updated installation instructions on Windows/Mac topics
- Removed references to Toolbox from the Install topics
- Updated the Install pages to add info on the new onboarding tutorial on startup

### CODE REVIEW

**Comment by usha-mandya** [comment]
Need to add the download URLs to the Windows Home topic when the build # is confirmed.

**Comment by nebuk89** [comment]
Looks good to me, we should get @simonferquel to have a look as someone who has installed it on Windows home 👍

**Comment by usha-mandya** [comment]
Sure. @simonferquel Could you PTAL?

I've tried the installation on a Windows Home test machine set up by Stefan. Let me know if there are any steps/caveats missing.

File: docker-for-windows/install-windows-home.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+---
+
+You can now install Docker Desktop on Windows 10 Home machines. Docker Desktop for WSL 2 Windows Home is a full version of Docker Desktop for Linux container development.
+
+This page contains information on installing Docker Desktop on Windows 10 Home. If you are looking for information about installing Docker Desktop on Windows 10 Pro, Enterprise, or Education, see [Install Docker Desktop on Windows](/install.md).
+
+[Download from Docker Hub](Add URL){:
+.button .outline-btn}
+
+Docker Desktop for WSL 2 Windows Home offers the following benefits:
```
Comment (simonferquel) [inline docker-for-windows/install-windows-home.md:14]: "Docker Desktop for WSL 2 Windows Home": is it a thing ?

File: docker-for-windows/install-windows-home.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+description: How to install Docker Desktop on Windows 10 Home
+keywords: Windows 10 Home, Home, Windows, install, download, run, Docker, local
+title: Install Docker Desktop on Windows Home
+---
+
+You can now install Docker Desktop on Windows 10 Home machines. Docker Desktop for WSL 2 Windows Home is a full version of Docker Desktop for Linux container development.
```
Comment (simonferquel) [inline docker-for-windows/install-windows-home.md:7]: "Docker Desktop for WSL 2 Windows Home": is it a thing ?

File: docker-for-windows/install-windows-home.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+### What's included in the installer
+
+The Docker Desktop installation includes [Docker Engine](/engine/index.md),
+Docker CLI client, [Docker Compose](/compose/index.md),
+[Notary](/notary/getting_started.md),
+[Kubernetes](https://github.com/kubernetes/kubernetes/),
+and [Credential Helper](https://github.com/docker/docker-credential-helpers/).
+
+Containers and images created with Docker Desktop are shared between all
+user accounts on machines where it is installed. This is because all Windows
```
Comment (simonferquel) [inline docker-for-windows/install-windows-home.md:55]: This is not true on Windows Home

Comment (simonferquel) [reply docker-for-windows/install-windows-home.md:55]: It is not the case with WSL2 on pro either

File: docker-for-windows/install-windows-home.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md:14]: Would it be better to say `Docker Desktop on Windows Home with the WSL 2 backend offers...`? Any other suggestions welcome :)

File: docker-for-windows/install-windows-home.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md]: Fixed. PTAL.

File: docker-for-windows/install-windows-home.md (hunk 3 above)
Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md]: Removed it from the Windows Home topic. Added a caveat in the Windows Pro topic that it is not possible to share containers and images when using WSL 2.

Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md]: Fixed.

File: docker-for-windows/install-windows-home.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md]: Updated as per our discussion in Slack.

**Comment by usha-mandya** [comment]
Converting to 'Draft' due to the new timelines.

File: docker-for-windows/install-windows-home.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+Nested virtualization scenarios, such as running Docker Desktop on a
+VMWare or Parallels instance might work, but there are no guarantees. For
+more information, see [Running Docker Desktop in nested virtualization scenarios](troubleshoot.md#running-docker-desktop-for-windows-in-nested-virtualization-scenarios).
+
+## Install Docker Desktop on Windows 10 Home
+
+1.  Double-click **Docker Desktop Installer.exe** to run the installer.
+
+    If you haven't already downloaded the installer (`Docker Desktop Installer.exe`), you can get it from
+    [**Docker Hub**](Add URL).
```
Comment (StefanScherer) [inline docker-for-windows/install-windows-home.md:65]: Add missing url, it's the same as in the Windows 10 Pro installation page.

```suggestion
    [**Docker Hub**](https://hub.docker.com/editions/community/docker-ce-desktop-windows/).
```

File: docker-for-windows/install-windows-home.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+Docker Desktop does not start automatically after installation. To start Docker Desktop, search for Docker, and select **Docker Desktop** in the search results.
+
+![search for Docker app](images/docker-app-search.png){:width="300px"}
+
+When the whale icon in the status bar stays steady, Docker Desktop is up-and-running, and is accessible from any terminal window.
+
+![whale on taskbar](images/whale-icon-systray.png)
+
+When the initialiation is complete, Docker Desktop launches the onboarding tutorial. The tutorial includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
```
Comment (StefanScherer) [inline docker-for-windows/install-windows-home.md:85]:
```suggestion
When the initialization is complete, Docker Desktop launches the onboarding tutorial. The tutorial includes a simple exercise to build an example Docker image, run it as a container, push and save the image to Docker Hub.
```

File: docker-for-windows/install-windows-home.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+  - Install Windows 10 Insider Preview build 19041 or higher.
+  - The following hardware prerequisites are required to successfully run Client
+Hyper-V on Windows 10:
+
+     - 64 bit processor with [Second Level Address Translation (SLAT)](http://en.wikipedia.org/wiki/Second_Level_Address_Translation)
+     - 4GB system RAM
+    - BIOS-level hardware virtualization support must be enabled in the
+    BIOS settings.  For more information, see
+    [Virtualization](troubleshoot.md#virtualization-must-be-enabled).
+  - Enable WSL 2 feature on Windows. For detailed instructions, refer to the[Microsoft documentation](https://docs.microsoft.com/windows/wsl/wsl2-install).
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-for-windows/install-windows-home.md]: Fixed.

**Comment by usha-mandya** [comment]
@StefanScherer PTAL

**Pull request merged**